### PR TITLE
Switched translations for US > GB

### PR DESCRIPTION
**Pull request by ScottIsAFool**
*sigh*

### CODE REVIEW

**Comment by ScottIsAFool** [comment]
This fixes issue #19

File: Emby.Mobile.Core/Strings/AppResources.en-GB.resx [hunk 1]
```diff
@@ -0,0 +1,1449 @@
...
+  </data>
+-->
+
+  <!--
+<data name="SysTrayGettingUpcoming" xml:space="preserve">
+    <value>Getting upcoming episodes...</value>
+  </data>
+-->
+
+  <data name="SysTrayLoadingProgrammes" xml:space="preserve">
```
Comment (7illusions) [inline Emby.Mobile.Core/Strings/AppResources.en-GB.resx]: Should we also change en-GB key naming to follow en-US?

Comment (ScottIsAFool) [inline Emby.Mobile.Core/Strings/AppResources.en-GB.resx]: Are you trying to make me cry? Does it overly matter?

File: Emby.Mobile.Core/Strings/AppResources.en-GB.resx [hunk 2]
```diff
@@ -0,0 +1,1449 @@
...
+    <value>Loading programmes...</value>
+  </data>
+  <data name="SysTrayCancellingProgramme" xml:space="preserve">
+    <value>Canceling programme...</value>
```
Comment (7illusions) [inline Emby.Mobile.Core/Strings/AppResources.en-GB.resx]: This is still en-GB translations

Comment (ScottIsAFool) [inline Emby.Mobile.Core/Strings/AppResources.en-GB.resx]: Oops, changed one part, missed the other

File: Emby.Mobile.Core/Strings/AppResources.en-GB.resx (hunk 1 above)
Comment (7illusions) [inline Emby.Mobile.Core/Strings/AppResources.en-GB.resx]: Ok then. :)

**Pull request merged**